Sources/Sentry/include/SentryFileManager.h

```diff
@@ -36,8 +36,8 @@ SENTRY_NO_INIT
 
 + (BOOL)createDirectoryAtPath:(NSString *)path withError:(NSError **)error;
 
+- (void)deleteOldEnvelopesFromAllSentryPaths;
```

Comment:
`m`: This doesn't have to be public. We can test it by calling the constructor.
```suggestion
```

Reply:
It's not public since the entire header isn't public. And I think testing it by calling the method is a whole lot easier?

Reply:
Testing it via the constructor has the advantage of testing it like the SDK code calls this method. Right now, your tests don't ensure the code gets executed when calling the constructor.

Reply:
I see your point, but we don't have to deal with async code and the flakiness that brings either 😬

Reply:
That's why you can use `SentryDispatchQueueWrapper` in the code and `TestSentryDispatchQueueWrapper` in the tests to avoid the flakyness.

Reply:
Done